internal/store/api_keys_access_list.go

```diff
@@ -91,72 +101,43 @@ func (s *Store) OrganizationAPIKeyAccessLists(orgID, apiKeyID string, opts *atla
 	case config.CloudService, config.CloudGovService:
 		result, _, err := s.client.(*atlas.Client).AccessListAPIKeys.List(context.Background(), orgID, apiKeyID, opts)
 		return result, err
-	case config.OpsManagerService, config.CloudManagerService:
+	case config.CloudManagerService, config.OpsManagerService:
 		result, _, err := s.client.(*opsmngr.Client).AccessListAPIKeys.List(context.Background(), orgID, apiKeyID, opts)
-		var target *atlas.ErrorResponse
-		if err != nil && errors.As(err, &target) {
-			// We keep supporting OM 4.2 and OM 4.4
-			if target.HTTPCode == resourceNotFound {
-				result, _, e := s.client.(*opsmngr.Client).WhitelistAPIKeys.List(context.Background(), orgID, apiKeyID, opts)
-				return fromWhitelistAPIKeysToAccessListAPIKeys(result), e
-			}
-		}
 		return result, err
 	default:
 		return nil, fmt.Errorf("%w: %s", errUnsupportedService, s.service)
 	}
 }
 
-// fromWhitelistAPIKeysToAccessListAPIKeys convert from atlas.WhitelistAPIKeys format to atlas.AccessListAPIKeys
-// We use this function with whitelist endpoints to keep supporting OM 4.2 and OM 4.4.
-func fromWhitelistAPIKeysToAccessListAPIKeys(in *atlas.WhitelistAPIKeys) *atlas.AccessListAPIKeys {
-	if in == nil {
-		return nil
-	}
-
-	out := &atlas.AccessListAPIKeys{
-		TotalCount: in.TotalCount,
-		Links:      in.Links,
-	}
-
-	results := make([]*atlas.AccessListAPIKey, len(in.Results))
-	for i, element := range in.Results {
-		results[i] = fromWhitelistAPIKeyToAccessListAPIKey(element)
+// CreateOrganizationAPIKeyWhitelist encapsulates the logic to manage different cloud providers.
+func (s *Store) CreateOrganizationAPIKeyWhitelist(orgID, apiKeyID string, opts []*atlas.WhitelistAPIKeysReq) (*atlas.WhitelistAPIKeys, error) {
```

Comment:
can we add a 
```suggestion
// CreateOrganizationAPIKeyWhitelist encapsulates the logic to manage different cloud providers.
//
// Deprecated: kept until 4.4 reaches EOL
func (s *Store) CreateOrganizationAPIKeyWhitelist(orgID, apiKeyID string, opts []*atlas.WhitelistAPIKeysReq) (*atlas.WhitelistAPIKeys, error) {
```

or something like it ?